webapp/portlet/src/main/webapp/vue-apps/activity-stream/components/activity/header/ActivityMobileHead.vue

```diff
@@ -1,5 +1,5 @@
 <template>
-  <v-list-item v-if="isSpaceStream">
+  <v-list-item v-if="isSpaceStream || !space" class="text-truncate">
```

Comment:
to add as computed **displaySpaceAvatar** = eXo.env.portal.spaceId !== '' || !this.space

Reply:
done!